docs/docs/migrate-data.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Migrating to v7: Migrating Data"
+---
+
+## Before you start
+
+Before you start the data migration process, make sure you have the following:
+
+- a tenant ID for the new v7 coral instance you have created (you can find this by inspecting the record in the `tenants` collection in your new mongo instance)
+- a site ID for the new v7 coral instance (inspect the one record in the `sites` collection)
```

Comment:
As far as I know, this is not known prior to performing the migration. When Wyatt and myself did the migration steps, we generated new UUID's for these values and used them as inputs to the `coral-importer` and `coral-import-mapper`.

When you were performing your steps, did you create a v7 instance first and use that tenant/site id instead?

Reply:
Nevermind, as you pointed out, SRE does this already. Never mind me and carry on, I'm just being overly cautious lol